arch/sim/Kconfig

```diff
@@ -378,6 +378,14 @@ config SIM_HCISOCKET
 		control of the device, but is abstracted from the
 		physical interface which is still handled by Linux.
 
+config SIM_BT_DEV_ID
```

Comment:
let's change to SIM_HCISOCKET_DEVID, because the additional transport(e.g. libusb) will be added in the future.